docs/page/lsp-org.md

```diff
@@ -0,0 +1,41 @@
+# Literate programming using LSP and org-mode(alpha)
+`lsp-mode` provides **experimental** support for running the language servers
+inside of [org-mode](https://orgmode.org/) source blocks. `lsp-mode` is doing
+that by obtaining the information about the source block and then translating
+the point to the LSP positions back and forth so the server is which so the
+server actually thinks that Emacs has opened the original file. In order that to
+work the files has to be present on the disk as well because the server expects
+to find them.
+
+
+``` org
+#+BEGIN_SRC python :tangle "python.py"
+print "Hello!"
+#+END_SRC
+```
+
+## What works
+* `lsp-mode` core features (finding references, going to definitions, completion, lenses, highlighting, etc)
+* `company-mode`
+* `flycheck`
+* `lsp-treemacs-symbols`
+* `lsp-treemacs-errros`
+
+## Known limitations
+* Incremental updates are performed by sending the whole
+  buffer each time(they simulate full updates).
+* Deleting content that spans across the source block and the content of the org
+  document is not handled.
+* Opening and editing the actual file from the disk might cause undefined
+  behavour because the server expects file to be open only once.
```

Comment:
```suggestion
* Incremental updates are performed by sending the whole buffer each time(they simulate full updates).
* Deleting content that spans across the source block and the content of the org document is not handled.
* Opening and editing the actual file from the disk might cause undefined behavior because the server expects file to be open only once.
```